pandas/core/window.py

```diff
@@ -831,13 +832,13 @@ def aggregate(self, arg, *args, **kwargs):
     @Appender(_shared_docs["sum"])
     def sum(self, *args, **kwargs):
         nv.validate_window_func("sum", args, kwargs)
-        return self._apply_window(mean=False, **kwargs)
+        return self._apply_window("roll_window_sum", **kwargs)
```

Comment:
IOW if we  blow aways _apply_window and move Rolling._apply to Window then we can unify the call signatures

Reply:
Is what you are suggesting using same apply function for both (Rolling and Window) or just same signature?

Reply:
former if possible latter if not